src/components/chronometer/chronometer-timer-state.tsx

```diff
@@ -0,0 +1,5 @@
+export enum TimerState {
```

Comment:
I'd move this enum to chronometer.tsx
I think it doesn't add much to keep it in it's own file.

Reply:
Stencil discourage having 2 exports in the same file**. If set as private, then components.d.ts would not compile. 

> @Component() should be the only export of the module. Numerous export statements within a component module
>            may cause undesirable bundling output, leading to unoptimized lazy loading. Create a new auxiliary `.ts`
>            file in order to export shared functionality.